example_dags/demo_parse_directory/ingest_models/adoption_center_1.sql

```diff
@@ -0,0 +1,7 @@
+---
+conn_id: snowflake_conn
+database: DWH_LEGACY
```

Comment:
This doesn't work when I try to run the tests locally.
Would there be a way of us not having this in the SQL header?

Reply:
@tatiana I've added in the ability to place "default" values in the render function to address this. I've also changed this DAG to work on postgres. Now the only one that won't work locally is the snowflake one, but that will still pass in CI